Makefile

```diff
@@ -132,9 +132,13 @@ install: manifests kustomize
 uninstall: manifests kustomize
 	$(KUSTOMIZE) build config/crd | kubectl delete -f -
 
+.PHONY: deploy-cert-manager
+deploy-cert-manager:
+	kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.1.0/cert-manager.yaml
```

Comment:
perhaps here we can wait that cert manageris up and running? That would benefical for the local dev cycle as well?

Reply:
Done in 176d73f10d5047ed3b2383d4136d23e693c874a5.